alica_engine/include/engine/modelmanagement/factories/PlaceholderMappingFactory.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "Factory.h"
+
+#include <yaml-cpp/yaml.h>
```

Comment:
nit:  Maybe better to forward declare class and only include in cpp file if possible?

Reply:
The create function takes a YAML::Node, so we need to include whatever file makes this class visible. I think yaml.h is the only one or I'm not sure, will have to check